### PR TITLE
Anton/RVAT-675_filter_for_camp_places_10000

### PR DESCRIPTION
**Pull request by romanukhaav**

### CODE REVIEW

File: src/test/java/rv/atqc/gttesting/filer_places/FilterPlacesChrome.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+	
+	private void setCampPlacesFilter(short select){
+		if (select == 1){
+			WebElement linkElement = wait.until(ExpectedConditions.visibilityOfElementLocated
+					(By.xpath("//*[@id='main']/div/place-list/div/div/div[1]/div[2]/div/div[3]/ul/li[1]/ul/li[1]")));
+			linkElement.click();
+		}		   
+	 }
+
+	private void setServicePlacesFilter(short select){
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/filer_places/FilterPlacesChrome.java]: line 88 - 116: a lot of same code. please think about how you can improve it.

**Review by TyshchenkoRoma — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -57,7 +57,7 @@
 		<dependency>
 			<groupId>org.seleniumhq.selenium</groupId>
 			<artifactId>selenium-java</artifactId>
-			<version>2.45.0</version>
+			<version>3.0.1</version>
```
Comment (dmatsev) [inline pom.xml:60]: merge from dev branch

File: src/test/java/rv/atqc/gttesting/filer_places/FilterPlacesChrome.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import io.github.bonigarcia.wdm.ChromeDriverManager;
+
+public class FilterPlacesChrome{
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/filer_places/FilterPlacesChrome.java:16]: please format the document... a lot of empty lines

**Review by dmatsev — APPROVED** [review]
approve with comments

**Review by MaxKovalets — APPROVED** [review]

**Pull request merged**